samples/kubeflow/README.md

```diff
@@ -53,6 +53,39 @@ cd samples/kubeflow/install/kubeflow
 
 cd -
 ```
+On an OpenShift 4+ cluster, follow [Kubeflow Documentation](https://www.kubeflow.org/docs/openshift/install-kubeflow/) to install Kubeflow with a few changes detailed below:
+```bash
+git clone https://github.com/opendatahub-io/manifests.git
+```
+
+Changes to `kfctl_openshift.yaml`:
+* Since we already have Istio installed, comment out `istio-crds` and `istio-install`
+* Comment out `cert-manager-crds` and `cert-manager` 
+* Comment out `seldon-core-operator` 
+
+A modified `kfctl_openshift.yaml` is provided in this repo.
```

Comment:
say that before listing the changes, by the time I read this statement it might be too late :)